csrc/device_lower/analysis/predicate_elimination.cpp

```diff
@@ -863,7 +863,18 @@ class PredicateChcker : public IterVisitor {
 } // namespace
 
 PredicateElimination::PredicateElimination(Fusion* fusion) {
-  traverseTo(fusion->outputs());
+  // TODO: Common code as in expr_sort.cpp.
```

Comment:
@wujingyue I want to move this to a common function in `fusion.cpp`. What do you think about merging `PointerArithmetic` and `Evaluate` in this PR itself?